src/components/icons/BulbIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const BulbIcon: ComponentWithAs<'svg', IconProps> = createIcon({
+  displayName: 'Car',
```

Comment:
Was this Car as displayName written on purpose? :)

Reply:
mistake :D